packages/dcos-integration-test/extra/test_networking.py

```diff
@@ -135,9 +170,14 @@ def test_vip(dcos_api_session, container: Container, vip_net: Network, proxy_net
         raise AssertionError('\n******************************************\n'.join(failure_stack))
 
 
-def setup_vip_workload_tests(dcos_api_session, container, vip_net, proxy_net):
+def setup_vip_workload_tests(dcos_api_session, pod, container, vip_net, proxy_net):
     if len(dcos_api_session.all_slaves) < 2:
         pytest.skip('must have more than one agent for this test!')
+    if pod:
+        if container == Container.DOCKER or container == Container.NONE:
+            pytest.skip('Not applicable for the pods')
+        if vip_net == Network.BRIDGE or proxy_net == Network.BRIDGE:
+            pytest.skip('Bridge mode network is not support in pods')
```

Comment:
This is not an appropriate way to handle the skipping of permutations that you do not intend to test at all right now. This is a huge swath of test cases that do not apply. Rather than selectively removing the bits of generate_vip_app_permutations that you do not want. Simply only generate the permutation that you do want in its own method